modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-account-impl/src/main/java/com/liferay/headless/commerce/admin/account/internal/resource/v1_0/AccountChannelEntryResourceImpl.java

```diff
@@ -1278,15 +1291,10 @@ private AccountChannelEntry _patchAccountChannelEntry(
 			int type)
 		throws Exception {
 
-		long commerceChannelId =
-			commerceChannelAccountEntryRel.getCommerceChannelId();
-
-		if (accountChannelEntry.getChannelId() != null) {
-			commerceChannelId = _getCommerceChannelId(
-				accountChannelEntry, commerceChannelAccountEntryRel.getType());
-		}
+		long commerceChannelId = _getCommerceChannelId(
```

Comment:
Why don't you just insert the code of the method inside the following supplier?:

Reply:
The idea is to reuse the `commerceChannelId` and not duplicate code. In 1312 line of the code, it uses the same variable.